### PR TITLE
Add factory method to create alert style bottom sheet

### PR DESCRIPTION
**Pull request by hackertronix**
Added a new method to create a simple re-usable BottomSheet as per new designs

## Screenshot

| Portrait | Landscape |
| ------ | ----- |
|Screenshot 2020-06-15 at 1 28 33 PM|Screenshot 2020-06-15 at 1 28 39 PM|

### CODE REVIEW

File: sample/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,3 +1,9 @@
 <resources>
     <string name="app_name">Sample</string>
+
```
Comment (hackertronix) [inline sample/src/main/res/values/strings.xml:3]: Added these for demonstration purposes

File: app/src/main/java/org/buffer/android/components/BottomSheetFactory.kt [hunk 2]
```diff
@@ -67,4 +70,31 @@ object BottomSheetFactory {
...
+    fun createSimpleAlertStyleBottomSheet(
+            context: Context,
+            @StringRes title: Int,
+            @StringRes message: Int,
+            @StringRes positiveButtonText: Int,
+            @StringRes neutralButtonText: Int,
+            @DrawableRes icon: Int,
+            postiveListener: View.OnClickListener? = null,
+            neutralListener: View.OnClickListener? = null
+
```
Comment (hitherejoe) [inline app/src/main/java/org/buffer/android/components/BottomSheetFactory.kt]: nit: blank space in constructor

**Review by hitherejoe — APPROVED** [review]
Looks good!

File: app/src/main/res/layout/view_simple_alert_sheet.xml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    android:background="@drawable/background_bottom_sheet"
+    android:orientation="vertical"
+    android:padding="24dp"
+    app:layout_behavior="android.support.design.widget.BottomSheetBehavior">
+
+    <ImageView
+        android:id="@+id/icon"
+        style="@style/AlertSheetImageStyle"
+        android:layout_width="wrap_content"
+        android:layout_height="114dp"
```
Comment (hitherejoe) [inline app/src/main/res/layout/view_simple_alert_sheet.xml:16]: this breaks when trying to use any image other than one sized like the IG reminder one @hackertronix . I'll open a JIRA ticket 👍

**Pull request merged**